library/cloudformation/index.js

```diff
@@ -3,11 +3,16 @@ const { CloudFormation } = require('aws-sdk');
 const apiVersions = require('../../apiVersions.json');
 const paramTools = require('./toolbox/parameter.tools');
 
-function CloudFormationWrapper() {
-  const cf = new CloudFormation({ apiVersion: apiVersions.CloudFormation });
+function CloudFormationWrapper({ accessKeyId, secretAccessKey, region }) {
+  const cf = new CloudFormation({
+    apiVersion: apiVersions.CloudFormation,
+    accessKeyId,
+    secretAccessKey,
+    region
+  });
 
   // -------------------------- API functions ---------------------------- //
-  async function changeTerminationProtection({ stackName, enableTerminationProtection }) {
+  async function changeTerminationProtection({ stackName, enableTerminationProtection } = {}) {
```

Comment:
Need to check that `stackName` is defined if the function allows an empty object.